designcompose/src/main/java/com/android/designcompose/CustomizationContext.kt

```diff
@@ -102,6 +107,7 @@ private fun Customization.clone(): Customization {
     c.text = text
     c.image = image
     c.imageWithContext = imageWithContext
+    c.brush = brush
```

Comment:
I think textFunction (which I probably missed) and brushFunction need to be cloned here also.

Reply:
Yep, good catch! Fixed!